<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
<launch>
	<arg name="freq" default="100"/>⏎
	<arg name="run_as_service" default="true" />
	<arg name="parent_frame" default="map"/>
	<arg name="heading_frame" 	default="subject_heading"/>
	<group ns="ik">
		<include file="$(find osrt_ros)/launch/ext_avg_quaternions_gait1992.launch">
			<arg name="tf_prefix" value="ik"/>
			<arg name="run_as_service" value="$(arg run_as_service)" />
		<arg name="parent_frame" value="$(arg parent_frame)"/>
		<arg name="heading_frame" value="$(arg heading_frame)"/>
		</include>

	</group>

	<!-- subject heading 3 with only offset so we can see what's going on. -->
	<node pkg="tf" type="static_transform_publisher" name="subject_heading_for_imus" args="-1.1 1.4 0  0 0 0 subject_heading subject_heading3 10" />




</launch>


